python/cuml/test/dask/test_random_forest.py

```diff
@@ -247,7 +253,9 @@ def test_rf_regression_dask_cpu(partitions_per_worker, client):
 @pytest.mark.parametrize('partitions_per_worker', [5])
 def test_rf_classification_dask_fil_predict_proba(partitions_per_worker,
                                                   client):
-    X, y = make_classification(n_samples=1000, n_features=30,
+    n_workers = len(client.scheduler_info()['workers'])
+
+    X, y = make_classification(n_samples=n_workers * 2700, n_features=30,
```

Comment:
Can this be n_workers * 500 or *1000 to be comparable to the old size?

Reply:
I could safely lower the number of samples to 1500. However, it's not possible to go much lower. The assertion for this test is `assert fil_mse <= sk_mse + 0.022`. It may be necessary to update that threshold as cuML and sklearn accuracies might have evolved a bit.